packages/nimble-components/src/table-column/text/template.ts

```diff
@@ -2,11 +2,18 @@ import { html } from '@microsoft/fast-element';
 import type { TableColumnTextCellRecord, TableColumnTextColumnConfig } from '.';
 import type { TableCellState } from '../../table/types';
 
+const setTitleWhenOverflow = (span: HTMLElement): void => {
+    if (span?.textContent?.trim() && span.offsetWidth < span.scrollWidth) {
```

Comment:
Is this line not causing lint errors? The type of `span` cannot be undefined based on the function signature, so you shouldn't need to the `?` in `span?`.

Reply:
Wasn't giving an error, but you're right. Removed.